source/compression/OP_COMPRESSED.rst

```diff
@@ -194,7 +194,7 @@ Compressor IDs
 Each compressor is assigned a predefined compressor ID.
 
 +-----------------+----------------+-------------------------------------------------------+
-| compressorId    | isMaster Value |  Description                                          |
+| compressorId    | hello Value    |  Description                                          |
```

Comment:
Should this be "handshake value" for consistency with the above, or was that intentionally not done to avoid a larger diff?

If you do go the route of a larger diff, you could also remove the leading space before "noop" in the row below.

Reply:
Done. And the phrase `Handshake Value` is just long enough to force me to reformat the entire table. 😛  So the `noop` fix got "added" for free.